modules/ossm-installing-multi-primary-single-network.adoc

```diff
@@ -0,0 +1,114 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-installing-multi-primary-single-network-mesh_{context}"]
+= Installing a multi-primary single-network mesh 
+:context: ossm-installing-multi-primary-single-network-mesh
+
+Install {istio} in the multi-primary single-network topology on two {ocp-product-title} clusters. 
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster. The East cluster is the primary cluster and the West cluster is the remote cluster.
+====
+
+You can adapt these instructions for a mesh spanning more than two clusters.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator on all of the clusters that comprise the mesh.
```

Comment:
I've just realized that we need another prerequisite for both single network topologies: both/all clusters must use the same network (there must be direct connectivity between the pods in both/all clusters). I think we can mention Submariner here. Check with the Submariner team to confirm.